submissions/EnF1NeeT/HTML-CSS_popup/index.html

```diff
@@ -0,0 +1,155 @@
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Storage.svg" alt="Storage" class="icon"> Storage</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Saved.svg" alt="Saved" class="icon"> Saved</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/recent.svg" alt="Recent" class="icon"> Recent</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Apps.svg" alt="Apps" class="icon"> Apps</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Locations.svg" alt="Locations" class="icon"> Locations</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Post.svg" alt="Posts" class="icon"> Posts</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/search.svg" alt="Search" class="icon"> Search</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/trash.svg" alt="Trash" class="icon"> Trash</a></li>
+                <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/calendar.svg" alt="Calendar" class="icon"> Calendar</a></li>
+              </ul>
+              <input type="checkbox" id="menu-check-more" class="input-check popup-item">
+              <label for="menu-check-more" class="check-more popup-item">↓</label>
+              <div class="main-menu-more">
+                <ul class="popup-grid-more list">
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/manage_account.svg" alt="Manage accounts" class="icon"> Manage accounts</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/security.svg" alt="Security" class="icon"> Security</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/payments.svg" alt="Payments" class="icon"> Payments</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/store.svg" alt="Store" class="icon"> Store</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/Settings.svg" alt="Settings" class="icon"> Settings</a></li>
+                  <li class="popup-item"><a href="#" class="item-name"><img src="./img/icons/logout.svg" alt="Logout" class="icon"> Logout</a></li>
+                </ul>
+              </div>
+            </div>
+          </li>
+          <li class="logo"><h1>Kottanspost</h1></li>
+          <li class="greeting"><h2>Hi, Kottan!</h2><img src="./img/account_circle_FILL0_wght400_GRAD0_opsz48.png" alt=""></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="posts">
+        <div class="post">
```

Comment:
For every post better to use the tag `article` for better semantics.

Reply:
Fixed in the last commit